tests/chainer_tests/functions_tests/activation_tests/test_randomized_leaky_relu.py

```diff
@@ -0,0 +1,74 @@
+import random
+import unittest
+
+import numpy
+
+import chainer
+from chainer import cuda
+from chainer import functions
+from chainer import gradient_check
+from chainer import testing
+from chainer.testing import attr
+from chainer.testing import condition
+
+
+@testing.parameterize(*testing.product({
+    'shape': [(3, 2), (5, 6)],
+    'dtype': [numpy.float16, numpy.float32, numpy.float64],
+}))
+class TestRReLU(unittest.TestCase):
+
+    def setUp(self):
+        # Avoid unstability of numeraical grad
+        self.x = numpy.random.uniform(-1, 1, self.shape).astype(self.dtype)
+        for i in numpy.ndindex(self.shape):
+            if -0.05 < self.x[i] < 0.05:
+                self.x[i] = 0.5
+        self.gy = numpy.random.uniform(-1, 1, self.shape).astype(self.dtype)
+        self.l = random.random()
+        self.u = random.random()
+        self.check_forward_options = {}
+        self.check_backward_options = {'dtype': numpy.float64}
+        if self.dtype == numpy.float16:
+            self.check_forward_options = {'atol': 1e-4, 'rtol': 1e-3}
+            self.check_backward_options = {
+                'dtype': numpy.float64, 'atol': 5e-4, 'rtol': 5e-3}
+
+    def check_forward(self, x_data):
+        x = chainer.Variable(x_data)
+        y = functions.randomized_leaky_relu(x, l=self.l, u=self.u, train=False)
```

Comment:
I think that I can't run testing if `self.r` is random. I will use  `y.creator.r`.
Thank you.